tests/audit-tests/src/test/java/utilities/kbv/Driver.java

```diff
@@ -0,0 +1,95 @@
+        //if this thread doesn't have driver - create it and add to pool
+        if (driverPool.get() == null) {
+
+//            if we pass the driver from terminal then use that one
+//           if we do not pass the driver from terminal then use the one properties file
+            String browser = ConfigurationReader.getBrowser();
+            LOGGER.info("ℹ️ browser is " + browser);
+            switch (browser) {
+                case "chrome":
+                    WebDriverManager.chromedriver().setup();
+                    driverPool.set(new ChromeDriver());
+                    break;
+                case "chrome-headless":
+                    WebDriverManager.chromedriver().setup();
+                    ChromeOptions chromeOptions = new ChromeOptions();
+                    chromeOptions.setHeadless(true);
+                    chromeOptions.addArguments("--no-sandbox");
+                    chromeOptions.addArguments("--disable-dev-shm-usage");
+                    driverPool.set(new ChromeDriver(chromeOptions));
+                    break;
+                case "firefox":
+                    WebDriverManager.firefoxdriver().setup();
+                    driverPool.set(new FirefoxDriver());
+                    break;
+                case "firefox-headless":
+                    WebDriverManager.firefoxdriver().setup();
+                    driverPool.set(new FirefoxDriver(new FirefoxOptions().setHeadless(true)));
+                    break;
+                case "ie":
+                    if (!System.getProperty("os.name").toLowerCase().contains("windows"))
+                        throw new WebDriverException("Your OS doesn't support Internet Explorer");
+                    WebDriverManager.iedriver().setup();
+                    driverPool.set(new InternetExplorerDriver());
+                    break;
+
+                case "edge":
+                    if (!System.getProperty("os.name").toLowerCase().contains("windows"))
+                        throw new WebDriverException("Your OS doesn't support Edge");
+                    WebDriverManager.edgedriver().setup();
+                    driverPool.set(new EdgeDriver());
+                    break;
+
+                case "safari":
+                    if (!System.getProperty("os.name").toLowerCase().contains("mac"))
+                        throw new WebDriverException("Your OS doesn't support Safari");
+//                        WebDriverManager.safaridriver().setup();
+//                        driverPool.set(new SafariDriver());
+                    WebDriverManager.getInstance(SafariDriver.class).setup();
+                    driverPool.set(new SafariDriver());
+                    break;
+            }
+        }
+        return driverPool.get();
+    }
+
+    public static void closeDriver() {
+        driverPool.get().quit();
+        driverPool.remove();
+        WebDriver driver=new ChromeDriver();
+        driver.close(); //closes the browser
+    }
```

Comment:
This is the only difference between this driver and the other one, and it's not used